test/system/src/test/java/io/pravega/test/system/ByteClientTest.java

```diff
@@ -182,5 +186,76 @@ public void byteClientTest() throws IOException {
         log.debug("Data correctly written/read from Stream: byte client test passed.");
     }
 
-    // End utils region
+    /**
+     * This test verifies the correctness of truncation in a ByteStream using {@link ByteStreamReader} and {@link ByteStreamWriter}.
+     */
+    @Test
+    public void byteClientTruncationTest() throws IOException {
+        log.info("byteClientTruncationTest:: with security enabled: {}", Utils.AUTH_ENABLED);
+
+        assertTrue("Creating stream", streamManager.createStream(SCOPE, STREAM_TRUNCATION, config));
+
+        @Cleanup
+        ConnectionFactory connectionFactory = new ConnectionFactoryImpl(Utils.buildClientConfig(controllerURI));
+        ControllerImpl controller = new ControllerImpl(ControllerImplConfig.builder()
+                                                                           .clientConfig(Utils.buildClientConfig(controllerURI)).build(),
```

Comment:
Nit: can we format these 3 lines properly please?

Reply:
fixed it.